Week3/homework/step4.js

```diff
@@ -1,6 +1,11 @@
-'use strict';
+"use strict";
 
 // Add your code here
+function createBase(val) {
+  return function(base) {
+    return console.log((base += val));
+  };
+}
 
 const addSix = createBase(6);
 
```

Comment:
✅ Nice job here.  The only thing is I would return the value (`base += val`) from the function itself, and `console.log` the result.  Normally, you will want to do something with the value from a function, and use it in further calculations.  Assuming it is only to be displayed is not a good idea.  
Basically, this is the same problem as in the first exercise with `foo` and `func`. 😄